Source/RapyutaSimulationPlugins/Public/Core/RRConversionUtils.h

```diff
@@ -118,18 +113,24 @@ class URRConversionUtils : public UBlueprintFunctionLibrary
 
         return 100.f * Output;
     }
-    FORCEINLINE static void VectorROSToUE(const double& InputX,
-                                          const double& InputY,
-                                          const double& InputZ,
-                                          double& OutputX,
-                                          double& OutputY,
-                                          double& OutputZ)
+    
+    FORCEINLINE static void VectorROSToUE(const FVector& Input, FVector& Output)
     {
-        OutputX = InputX * 100.f;
-        OutputY = -InputY * 100.f;
-        OutputZ = InputZ * 100.f;
+        Output.Set(Input.X * 100.f, -Input.Y * 100.f, Input.Z * 100.f);
     }
 
+    // FORCEINLINE static void VectorROSToUE(const double& InputX,
+    //                                       const double& InputY,
+    //                                       const double& InputZ,
+    //                                       double& OutputX,
+    //                                       double& OutputY,
+    //                                       double& OutputZ)
+    // {
+    //     OutputX = InputX * 100.f;
+    //     OutputY = -InputY * 100.f;
+    //     OutputZ = InputZ * 100.f;
+    // }
```

Comment:
```suggestion
#if RAPYUTA_SIM_DEBUG
    // This would be obviated in UE5, in which FVector is of double by default
    FORCEINLINE static void VectorROSToUE(const double& InputX,
                                          const double& InputY,
                                          const double& InputZ,
                                          double& OutputX,
                                          double& OutputY,
                                          double& OutputZ)
    {
        OutputX = InputX * 100.f;
        OutputY = -InputY * 100.f;
        OutputZ = InputZ * 100.f;
     }
#endif
```